src/app/containers/FeaturesAnalysis/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, node } from 'prop-types';
+import SectionLabel from '@bbc/psammead-section-label';
+import { C_GHOST } from '@bbc/psammead-styles/colours';
+import styled from 'styled-components';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+
+import featuresAnalysis from '#pages/StoryPage/featuresAnalysis.json';
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GridItemConstrainedLarge } from '#lib/styledGrid';
+import StoryPromo from '../StoryPromo';
+
+const Wrapper = styled(GridItemConstrainedLarge)`
+  margin-bottom: ${GEL_SPACING_DBL};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+  /* z-index needs explicitly set as the psammead section label component uses negative z-indices */
+  z-index: 0;
```

Comment:
I'm still unsure why a z-index is being set here. Please can you clarify why this is needed?

Reply:
This was copied along with the code from `CpsRelatedContent` container. I'm not sure that it's necessary for this component though.

Reply:
I've removed the `z-index`.